src/integ_test_resources/android/sdk/integration/cdk/README.md

```diff
@@ -20,25 +20,35 @@ npm install -g aws-cdk
 Ensure that you have [credentials in your environment sufficient to run
 the CDK](https://docs.aws.amazon.com/cdk/latest/guide/getting_started.html#getting_started_credentials).
 
-Install CDK dependencies:
+Install CDK dependencies from the root directory of the repo:
 ```console
 # Best to do this in a virtualenv
-pip install -r requirements.txt
+pip3 install -r requirements.txt
 ```
 
-Generate a Pinpoint stack template at
+**Export credentials and region:**
+
+Export appropriate role credentials to your local shell.
+
+Also, set the AWS region to us-east-1:
```

Comment:
Is this universally applicable? We run the iOS tests in us-west-2, and at least in theory should support any region (subject to service availability, of course)

Reply:
Ah, we should clarify this point. The Android tests are mainly hardcoded in us-east-1, thus the above line. (*grumblegrumblegrumble*)

Reply:
Since we currently have this README specifically under the Android directory I think it should be fine, right? If not, I'll be happy to submit another PR.